src/HubBootstrapper.php

```diff
@@ -0,0 +1,180 @@
+                        SdkIntegration\FatalErrorListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    // We also remove the default request integration so it can be readded
+                    // after with a Yii3 specific request fetcher. This way we can resolve
+                    // the request from Yii3 instead of constructing it from the global state
+                    return ! ($integration instanceof
+                        SdkIntegration\RequestIntegration);
+                }
+            );
+
+            $integrations[] = new SdkIntegration\RequestIntegration(
+                new YiiRequestFetcher($this->container)
+            );
+        }
+
+        return array_merge($integrations, $userIntegrations);
+    }
+
+    /**
+     * Resolve the integrations from the user configuration with the container.
+     *
+     * @return SdkIntegration\IntegrationInterface[]
+     */
+    private function resolveIntegrationsFromUserConfig(): array
+    {
+        // Default Sentry SDK integrations
+        $integrations = [
+            new Integration(),
+            new Integration\ExceptionContextIntegration(),
+        ];
+
+        $integrationsToResolve = $this->configuration->getIntegrations();
+
+        $enableDefaultTracingIntegrations = isset(
+            $this->configuration
+                ->getTracing()['default_integrations']
+        )
+            ? (bool)$this->configuration->getTracing()['default_integrations']
+            : true;
+
+        if (
+            $enableDefaultTracingIntegrations
+            && $this->configuration->couldHavePerformanceTracingEnabled()
+        ) {
+            $integrationsToResolve = array_merge(
+                $integrationsToResolve,
+                self::DEFAULT_INTEGRATIONS
+            );
+        }
+        /** @psalm-suppress MixedAssignment */
+        foreach ($integrationsToResolve as $userIntegration) {
+            if (
+                $userIntegration instanceof
+                SdkIntegration\IntegrationInterface
+            ) {
+                $integrations[] = $userIntegration;
+            } elseif (is_string($userIntegration)) {
+                /** @psalm-suppress MixedAssignment */
+                $resolvedIntegration = $this->container->get($userIntegration);
+
+                if (
+                    !$resolvedIntegration instanceof
+                        SdkIntegration\IntegrationInterface
+                ) {
+                    if (is_array($resolvedIntegration)) {
+                        $value = 'array';
+                    } elseif (is_object($resolvedIntegration)) {
+                        $value = $resolvedIntegration::class;
+                    } elseif (null === $resolvedIntegration) {
+                        $value = 'null';
+                    } else {
+                        $value = (string)$resolvedIntegration;
+                    }
```

Comment:
Perhaps can be simplified with something like `get_debug_type()`. A similar case can be found in validator.